.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
```

Comment:
My editor was complaining about this not being here, so I went ahead and added it. I'm happy to move it to another PR  if needed.